tasks/post-to-cmr/schemas/input.json

```diff
@@ -1,14 +1,46 @@
 {
   "title": "PostToCmrInput",
   "description": "Describes the input expected by the post-to-cmr task",
-  "type": "array",
-  "items": {
+  "type": "object",
+  "required": [
+    "granules",
+    "inputFiles"
+  ],
+  "properties": {
     "granules": {
       "type": "array",
+      "description": "Array of all granules",
       "items": {
-        "type": "string",
-        "description": "list of s3 uris"
+        "type": "object",
+        "required": ["granuleId", "files"],
+        "properties": {
+          "granuleId": {
+            "type": "string"
+          },
+          "files": {
+            "type": "array",
+            "items": {
+              "type": "object",
+              "required": ["name"],
+              "properties": {
+                "name": {
+                  "type": "string"
+                },
+                "filename": {
+                  "type": "string"
+                },
+                "bucket": {
+                  "type": "string"
+                }
+              }
+            }
+          }
+        }
       }
+    },
+    "inputFiles": {
```

Comment:
If the files are in the granules.files already, we don't need `inputFiles`

Reply:
I see what you mean. Made the change